Docker/itk-base/Dockerfile

```diff
@@ -16,7 +17,11 @@ RUN curl -O https://cmake.org/files/v3.4/cmake-3.4.3.tar.gz && \
     cd /tmp/cmake-build && \
     ../cmake-3.4.3/bootstrap && \
     make -j$(nproc) && \
-    ./bin/cmake -DCMAKE_BUILD_TYPE:STRING=Release . && \
+    cd .. && \
```

Comment:
> Ideally, the official CMake binaries should be built with manylinux so that we can run on most of linux distrib.

All manylinux does is use an old linux environment (CentOS 5) to try to give the best compatibility possible.  The CMake binaries effectively already do this as they're built on a very old Debian distro to give maximum forward compatibility.  So in that regard, just using the official Linux binaries should be fine in which case the original need to build a new cmake here could have probably been avoided in the first place.

However, this turns on OpenSSL support which is not present in the official binaries due to export control issues.  So even if the official cmake binaries worked in the image as is (which I suspect they probably do) you still need to build it from source here to get SSL support.